torchsde/_core/base_sde.py

```diff
@@ -102,6 +65,15 @@ def __init__(self, sde):
         }[sde.noise_type]
         # TODO: Assign `gdg_jacobian_contraction`.
 
+    def f(self, t, y):
+        return self._base_sde.f(t, y)
+
+    def g(self, t, y):
+        return self._base_sde.g(t, y)
```

Comment:
What's the rationale for this change? They were directly assigned to remove the overhead of the extra indirection, same as we've been doing elsewhere.